sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md

```diff
@@ -0,0 +1,73 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spotenantrenamesiteprioritization
+applicable: SharePoint Online
+title: Set-SPOTenantRenameSitePrioritization
+author: AnfraMsft
+ms.author: anfra
+ms.reviewer: anushmag
+manager: anushmag
+schema: 2.0.0
+ms.topic: reference
+---
+
+# Set-SPOTenantRenameSitePrioritization
+
+## SYNOPSIS
+This cmdlet allows prioritization of a site for early execution, as part of [Advanced Tenant Rename](https://aka.ms/advancedtenantrename).
+
+## SYNTAX
+
+```
+Set-SPOTenantRenameSitePrioritization -SiteUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+As part of [Advanced Tenant Rename](https://aka.ms/advancedtenantrename), organizations can prioritize up to 4,000 sites in their scheduled rename.
+
+Prioritizing a site means that it will be queued for initial execution among all other sites in your organization as part of the overall rename operation.
+
+To run this cmdlet, you need SharePoint admin permissions. Also, it is possible to start prioritizing sites only once the rename operation has been scheduled using the [Start-SPOTenantRename](https://learn.microsoft.com/powershell/module/sharepoint-online/start-spotenantrename?view=sharepoint-ps) cmdlet.
```

Comment:
>To run this cmdlet, you need SharePoint admin permissions.  [](http://example.com/codeflow?start=0&length=59)

remove this, then add this line at the end of this block:

`You must be a SharePoint or Global Administrator to run this cmdlet.`